services/cms/src/components/editors/GutenbergEditor.tsx

```diff
@@ -70,8 +71,23 @@ const GutenbergEditor: React.FC<GutenbergEditorProps> = ({
   }, [courseId])
 
   const handleChanges = (newContent: BlockInstance[]): void => {
-    console.log(newContent)
-    onContentChange(newContent)
+    const modfiedNewContent = newContent.map((block) => {
```

Comment:
do this in this component/in the parent of this component before we pass the content to BlockEditorProvider